ui/selectors/selectors.js

```diff
@@ -340,6 +341,13 @@ export function getSelectedInternalAccount(state) {
   return state.metamask.internalAccounts.accounts[accountId];
 }
 
+export function isSelectedInternalAccountEth(state) {
```

Comment:
Should we start an `accounts` selectors directory instead of bloating this file? We can expose all the selectors at an index file in the root

Reply:
Sounds like a good idea! Lemme do that. I'll only add this one for now, we'll move other account related selector later.